stress/shell/src/main/java/alluxio/stress/cli/StressJobServiceBench.java

```diff
@@ -316,7 +316,7 @@ private void runDistributedLoad(String dirPath) throws AlluxioException, IOExcep
             new AlluxioURI(dirPath), numReplication, new HashSet<>(), new HashSet<>(),
             new HashSet<>(), new HashSet<>(), false, false);
       } finally {
-        mResult.incrementNumSuccess((long) cmd.getCompletedCount() * mParameters.mBatchSize);
+        mResult.incrementNumSuccess(cmd.getCompletedCount());
```

Comment:
to avoid the same problem in a future iteration, i may suggest to update the method name to reflect what is being counted, ex. getCompletedFileCount() vs getCompletedOpsCount()